python/tk_multi_reviewsubmission/renderer.py

```diff
@@ -272,7 +272,11 @@ def run(self):
 
         if p.returncode != 0:
             output_str = '\n'.join(output_lines)
-            self.subproc_error_msg = output_str.split('[RETURN_STATUS_DATA]')[1]
+            try:
+                self.subproc_error_msg = output_str.split('[RETURN_STATUS_DATA]')[1]
+            except IndexError:
+                # if error has not been formatted by the subprocess, get all available info
+                self.subproc_error_msg = output_str
```

Comment:
If error occurs outside the render_in_nuke() function in the subprocess, output string does not contain 'RETURN_STATUS_DATA'.
We get an IndexError here and the actual stderr message does not get propagated.